src/charm.py

```diff
@@ -74,6 +74,12 @@ def __init__(self, *args):
         self.framework.observe(self.on.config_changed, self._update_all_metrics_consumers)
         # Ensure we refresh scrape jobs on charm upgrade
         self.framework.observe(self.on.upgrade_charm, self._update_all_metrics_consumers)
+        # Initial charm setup
+        self.framework.observe(self.on.install, self._on_start)
+
+    def _on_start(self, _) -> None:
+        """Do any initial charm startup operations."""
+        self.unit.set_workload_version("n/a")
```

Comment:
Just for clarity. Why do not we name this method `_on_install` since the event workflow is:
 
- install -> config_changed -> start

Reply:
Oops I was testing something and forgot to change it back.